Misc/NEWS.d/next/IDLE/2018-03-05-01-29-05.bpo-32984.NGjgT4.rst

```diff
@@ -0,0 +1,6 @@
+Set __file__ while running a startup file. Like Python, IDLE optionally runs
+1 startup file in the Shell window before presenting the first interactive
+input prompt.  For IDLE, -s runs a file named in environmental variable
+IDLESTARTUP or PYTHONSTARTUP; -r file runs file.  Python sets __file__ to
+the startup file name before running the file and unsets it before the first
+prompt.  IDLE now does the same when run normally, without the -n option.
```

Comment:
Set ``__file__`` while running a startup file.  Like Python, IDLE optionally
runs one startup file in the Shell window before presenting the first
interactive input prompt.  For IDLE, ``-s`` runs a file named in
environmental variable :envvar:`IDLESTARTUP` or :envvar:`PYTHONSTARTUP`;
``-r`` file runs file.  Python sets ``__file__`` to the startup file
name before running the file and unsets it before the first prompt.  IDLE now
does the same when run normally, without the ``-n`` option.

Reply:
Thanks.  by 'editing', I could see and copy the markup (and maybe learn better).  I changed one clause to ``-r file`` runs ``file``,  which I can preview here ;-).